### PR TITLE
arangodb 2.4.0

### PR DESCRIPTION
**Pull request by fceller**
Upgraded to ArangoDB 2.4.0

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Fails to build, looks like the `inreplace` may be no longer needed: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19879/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_arangodb/

**Comment by fceller** [comment]
Fixed inreplace

File: Library/Formula/arangodb.rb [hunk 1]
```diff
@@ -27,8 +27,8 @@ def install
     # Bundled V8 tries to build with a 10.5 deployment target,
     # which causes clang to error out b/c a 10.5 deployment target
     # and -stdlib=libc++ are not valid together.
-    inreplace "3rdParty/V8/build/standalone.gypi",
-      "'mac_deployment_target%': '10.5',",
+    inreplace "3rdParty/V8-3.29.59/build/standalone.gypi",
```
Comment (mikemcquaid) [inline Library/Formula/arangodb.rb]: Are you sure this is still needed e.g. does it fail without it?

**Comment by fceller** [comment]
Ok, it also compiles without the "inreplace".

**Pull request closed without merging**